src/utils/url.ts

```diff
@@ -29,8 +29,7 @@ export const venueRoomUrl = (venue: WithId<AnyVenue>, roomTitle: string) => {
 };
 
 export const isExternalUrl = (url: string) =>
-  url.includes("http") &&
-  window.location.host !== new URL(getRoomUrl(url)).host;
+  window.location.host !== new URL(url).host;
```

Comment:
If `new URL()` fails to parse it will throw an exception. There's some related commented out code for this in this file because we found it was breaking for some reason.. I think for this function though, wrapping it in a `try`/`catch`, and then 'falling back' to a sane default if we can't parse the URL might be a good idea?

My gut feel is that default would be opening in the same window (in case it's a relative URL, etc).. and maybe using `Bugsnag.notify` to give us that information as well (so that we are capturing data to know what is broken, and ensure we come back to validate/fix it)

- https://docs.bugsnag.com/platforms/javascript/reporting-handled-errors/

You can customize the event severity when using this, so since we should fall back to a 'sane default', I think it would be ok to drop the severity for these reported errors to warn or even info:

- https://docs.bugsnag.com/platforms/javascript/reporting-handled-errors/#customizing-diagnostic-data

[OOS/Future] The 'proper' higher level fix is to improve our validation wherever users can enter URLs to ensure that it can only ever be correct, + updating/sanitising all the existing DB data as required to ensure that aligns with our newly chose validation rules as well.

Reply:
Try/catch block has been added with a simple Bugsnag.notify(error) and severity `info`
I'm not sure if this is enough, you seem to be more experienced with this tool, if you have any other suggestions for the handling of the error, they are welcome.

Reply:
Looks good to me. I think @thecadams mentioned on one of my similar PR's the other day that where possible we should check the type of error, and only catch the ones we are explicitly caring about, letting the others get thrown higher again. So in this case, that would be catching URL parsing errors, and otherwise throwing them higher.

I think that was doing something like:

```
catch (e) {
  if (e.name === 'TypeError') {
    // bugsnag.notify
    // return false
  }
 throw e; // Error wasn't one we wanted to handle, let it bubble up higher.
}
```